lib/response-builder.ts

```diff
@@ -29,6 +29,7 @@ function buildResponse(
     plugin,
     package: pkg,
     manifestFiles,
+    scanResults: [], // TODO
```

Comment:
what is the todo here?

Reply:
```scanResults``` are expected to actually have things in them eventually